File: src/main/java/com/gitlab/daring/image/util/ExtStringUtils.java
package com.gitlab.daring.image.util;

import one.util.streamex.StreamEx;

import static org.apache.commons.lang3.StringUtils.split;

public class ExtStringUtils {

    public static StreamEx<String> splitAndTrim(String str, String sepChars) {
        return StreamEx.of(split(str, sepChars))
            .map(String::trim).remove(String::isEmpty);
    }

    public static String[] trimAll(String[] ss) {
        for (int i = 0; i < ss.length; i++) ss[i] = ss[i].trim();
        return ss;
    }

    private ExtStringUtils() {
    }
}

File: src/main/kotlin/com/gitlab/daring/image/util/ExtStringUtils.kt
package com.gitlab.daring.image.util

import org.apache.commons.lang3.StringUtils.split

object ExtStringUtils {

    fun splitAndTrim(str: String, sepChars: String): List<String> {
        return split(str, sepChars).map(String::trim).filter(String::isNotEmpty)
    }

}